core/designsystem/src/main/java/com/iteneum/designsystem/components/Text.kt

```diff
@@ -0,0 +1,205 @@
+@param color The color of the body text.
+@param onClick Optional high-order function that defines the click behavior when the body text is clicked.
+@author Irving Ulises Gonzalez Uscanga
+
+[LPBodyMedium] is a composable function that displays a medium-sized body text with the specified label, color, and optional onClick event. It can also be clickable and trigger an onClick event.
+@param modifier Modifier is used to apply custom styles and parameters to the body text, such as the width and height of the component.
+@param label The text label to be displayed as the body text.
+@param color The color of the body text.
+@param onClick Optional high-order function that defines the click behavior when the body text is clicked.
+@author Irving Ulises Gonzalez Uscanga
+
+[LPBodySmall] is a composable component that displays a body text with a small style in the user interface.
+@param modifier Modifier to specify the style and parameters of the component, such as the width of the component.
+@param label Text to be displayed in the user interface.
+@param color Text color.
+@param onClick Higher-order function that allows assigning functionality to the component when it is clicked.
+@author Irving Ulises Gonzalez Uscanga
+
+[LPSubTitleLarge] is a composable component that displays a subtitle text with a large style in the user interface.
+@param modifier Modifier to specify the style and parameters of the component, such as the width of the component.
+@param label Text to be displayed in the user interface.
+@param color Text color.
+@param onClick Higher-order function that allows assigning functionality to the component when it is clicked.
+@author Irving Ulises Gonzalez Uscanga
+
+[LPSubTitleMedium] is a composable component that displays a subtitle text with a medium style in the user interface.
+@param modifier Modifier to specify the style and parameters of the component, such as the width of the component.
+@param label Text to be displayed in the user interface.
+@param color Text color.
+@param onClick Higher-order function that allows assigning functionality to the component when it is clicked.
+@author Irving Ulises Gonzalez Uscanga
+
+[LPSubTitleSmall] is a composable component that displays a subtitle text with a small style in the user interface.
+@param modifier Modifier to specify the style and parameters of the component, such as the width of the component.
+@param label Text to be displayed in the user interface.
+@param color Text color.
+@param onClick Higher-order function that allows assigning functionality to the component when it is clicked.
+@author Irving Ulises Gonzalez Uscanga
+
+*/
+
+@Composable
+fun LPTitleLarge(
+    modifier: Modifier = Modifier,
+    label: String,
+    color: Color,
+    textAlign: TextAlign? = TextAlign.Center
+) {
+    Text(
+        text = label,
+        color = color,
+        style = LPTypography.titleLarge,
+        modifier = modifier, textAlign = textAlign)
```

Comment:
Please move this code in the next line `textAlign = textAlign)` same for the next functions